p2p/pre2p/module.go

```diff
@@ -77,6 +77,17 @@ func (m *p2pModule) GetBus() modules.Bus {
 func (m *p2pModule) Start() error {
 	log.Println("Starting network module")
 
+	m.
+		GetBus().
```

Comment:
I feel like `m.GetBus().` should be the whole first line.

Looks like too much whitespace.